SpatialConnect/SCAuthProtocol.h

```diff
@@ -30,6 +30,12 @@
 - (BOOL)authenticate:(NSString *)u password:(NSString *)p;
 
 /*!
+ *  @brief refreshes a valid token after expiration
+ *
+ */
+- (BOOL)refreshToken;
```

Comment:
Do you think we need to add this to the interface/protocol?  I'm thinking refreshing the token is specific to an implementation of SCAuthProtocol that uses OAuth to refresh the token.

Reply:
Thats cool, we can make it specific to the exchange auth